dotnetapp-3.5/dotnetapp-3.5/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using static System.Console;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        string message = "Dotnet-bot: Welcome to using .NET Framework!";
+
+        if (args.Length > 0)
+        {
+            message = System.String.Join(" ", args);
```

Comment:
Fixed in https://github.com/Microsoft/dotnet-framework-docker-samples/commit/907db099980fa4231ca56786d60d7066a0b0d663